chain/indexer/src/config.rs

```diff
@@ -18,6 +20,27 @@ pub struct IndexerConfig {
     /// Default is `true`.
     #[default(true)]
     pub fast_sync: bool,
+
+    /// Whether to perform logs snapshot download on startup.
+    /// When enabled, it allows for quicker indexing from scratch.
+    ///
+    /// Default is `false`.
+    #[default(false)]
+    pub enable_logs_snapshot: bool,
+
+    /// URL to download logs snapshot from.
+    /// This should point to a publicly accessible tar.gz file containing
+    /// the SQLite logs database files.
+    ///
+    /// Default is None
+    #[default(None)]
+    pub logs_snapshot_url: Option<String>,
```

Comment:
`validator::Validate` could be implemented to make sure:
1. that the URL indeed is a url format
2. that the expected file format is correct (ends with tar.gz...)

Reply:
done